examples/subprocess-output/subprocess-output.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+const React = require('react');
+const {render, Text, Box} = require('../../build');
```

Comment:
You don't have to explicitly include `build` folder, because it's already set as the `main` folder in `package.json`.

```suggestion
const {render, Text, Box} = require('../..');
```